core/serialization.go

```diff
@@ -225,7 +225,7 @@ func (h *BlobHeader) GetQuorumBlobParamsHash() ([32]byte, error) {
 		qbp[i] = quorumBlobParams{
 			QuorumNumber:                 uint8(q.QuorumID),
 			AdversaryThresholdPercentage: uint8(q.AdversaryThreshold),
-			QuorumThresholdPercentage:    uint8(q.QuorumThreshold),
+			QuorumThresholdPercentage:    uint8(q.ConfirmationThreshold),
```

Comment:
Are we renaming `QuorumThreshold` to `ConfirmationThreshold` in contracts as well? or just offchain?

Reply:
yes name change made on contracts as well